_unittest/test_98_Icepak.py

```diff
@@ -1452,3 +1452,10 @@ def test_73_conducting_plate(self):
         )
         with pytest.raises(AttributeError):
             self.aedtapp.assign_conducting_plate_with_conductance([box_face.id, "surfPlateTest"])
+
+    def test_74_csv_import(self):
+        self.aedtapp.insert_design("csv_import")
+        csv_file = os.path.join(
+            local_path, "../_unittest/example_models", test_subfolder, "cylinder_geometry_creation.csv"
+        )
+        assert self.aedtapp.create_geometry_from_csv(csv_file=csv_file, geo_type="cylinder")
```

Comment:
```suggestion
        assert self.aedtapp.create_geometry_from_csv(csv_file=csv_file, geo_type="cylinder")
        assert not self.aedtapp.create_geometry_from_csv(csv_file=csv_file, geo_type="cube")
```
So we can check if error is raised.